pkgs/test/mono_pkg.yaml

```diff
@@ -1,18 +1,14 @@
 dart:
-  - dev
+  - be/raw/latest
```

Comment:
Why this change?

Reply:
Oh this can switch back to dev now, good catch